Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/SubmissionSettings.kt

```diff
@@ -35,25 +36,54 @@ class SubmissionSettings @Inject constructor(
         context.getSharedPreferences("submission_localdata", Context.MODE_PRIVATE)
     }
 
+    val registrationToken = prefs.createFlowPreference<String?>(
+        key = TEST_REGISTRATION_TOKEN,
+        defaultValue = null
+    )
+
+    var initialTestResultReceivedAt: Instant?
+        get() = prefs.getLong(TEST_RESULT_RECEIVED_AT, 0L).let {
```

Comment:
to keep your high code quality, dont repeat yourself :octocat: 
..get() = ....

Reply:
made the code a bit more DRY